jscomp/others/js_json.ml

```diff
@@ -108,10 +108,12 @@ let decodeNull json : _ Js.null option =
 external parseExn : string -> t = "parse"
   [@@bs.val] [@@bs.scope "JSON"]
 
-external stringifyAny : 'a -> string option =
+external stringifyAnyExn : 'a -> string option =
 "stringify" [@@bs.val]  [@@bs.scope "JSON"]
 (* TODO: more docs when parse error happens or stringify non-stringfy value *)
```

Comment:
I think it'd be good to cover this TODO as well. Now that we're explicitly handling exceptions it would be nice to detail why things look like they look and when one can expect throwing.

Reply:
Does the doc means the doc comments in `*.mli`?